src/main/java/dev/slimevr/gui/VRServerGUI.java

```diff
@@ -222,7 +222,9 @@ public void mouseClicked(MouseEvent e) {
 					}
 				});
 				setFloorClipEnabled(
-					(boolean) server.config.getProperty("legTweaks.floorClip")
+					server.humanPoseProcessor
+						.getSkeletonConfig()
+						.getToggle("Floor Clip")
```

Comment:
Use the enum instead of a hardcoded string
```suggestion
						.getToggle(SkeletonConfigToggle.FLOOR_CLIP)
```